src/components/messages/broadcast.js

```diff
@@ -0,0 +1,68 @@
+import React, {useEffect, useState} from "react";
+import MessageInput from "./message-input";
+import * as service from "../../services/auth-service";
+import * as userService from "../../services/users-service";
+import {useNavigate} from "react-router-dom";
+import "./index.css";
+import MultiSelectUsers from "./multi-select-users";
+import * as messageService from "../../services/messages-service";
+
+/**
+ * Component to represent Broadcast Messages screen
+ * @returns {JSX.Element} React component
+ * @constructor
+ */
+const Broadcast = () => {
+  const navigate = useNavigate();
+  const [users, setUsers] = useState([]);
+  const [selectedUsers, setSelectedUsers] = useState([]);
+
+  useEffect(() => {
+    service.profile().then(me => {
+      userService.findAllUsers().then(userList => {
+        setUsers(userList.filter(user => user._id !== me._id));
+      });
+    }).catch(e => {
+      navigate('/login', {
+        state: {
+          redirect: '/messages',
+        }
+      });
+    })
+  }, []);
+
+  /**
+   * Sends the broadcast message
+   * @param message the message
+   * @param attachment the attachment if present
+   */
+  const send = (message, attachment) => {
```

Comment:
Should we also have validation to check if the document is of any specific type? Or should we just allow files with any extensions?

Reply:
I don't think allowing a specific type of files will be of any benefit to us. The way we have built the components, we can keep this generic, more like a document sharing feature than just an image sharing feature